### DISCUSSION TITLE
wrong chromedriver path setup

### DISCUSSION THREAD

**Discussion by peterninetythree** [aahnik/wappdriver#43] [Q&A] [answered]
Hi and thanks for this!
I just made a typo in the path to chromedriver at first execution, so now I'm getting "WappDriver Error  : ChromeDriver not loaded"
kinda going crazy to fix that...

**Comment by aahnik** [comment, marked as answer]
# Set Chrome Driver Path

When you will use `wappdriver` for the first time, it will ask you to input the path of Chrome Driver Executable in your system.

You can set the path programmatically

```python
from wappdriver import set_chrome_driver_path
path = '/home/aahnik/Downloads/chrome_driver'
set_chrome_driver_path(path)
```

[visit source](https://aahnik.github.io/wappdriver/usage/04_advanced/#set-chrome-driver-path)

<details>
<summary> 👉 Click here to watch an ad </summary>

[![promoted](https://raw.githubusercontent.com/aahnik/aahnik/master/promoted/general.gif)](https://aahnik.github.io/aahnik/promoted/general.html)

</details>

**Reply by peterninetythree** [reply]
That is excellent dude! Thanks a lot, all fixed works marvellously :)
